### DISCUSSION TITLE
アドベントカレンダーのスパム

### DISCUSSION THREAD

**Discussion by simon-ritchie** [increments/qiita-discussions#75] [ヘルプ] [answered]
アドベントカレンダーのプログラミング言語などのカテゴリでスパムで多くのカレンダーが生成されているようです！

![image](https://user-images.githubusercontent.com/25178045/144741329-80ba00c0-e5e8-4125-b332-abe32aeaf0c8.png)

**Comment by kyntk** [comment]
@simon-ritchie ご報告ありがとうございます! 対応します!

**Comment by kskwtnk** [comment, marked as answer]
@simon-ritchie 

こちら、ご報告ありがとうございました！

対応方針固まりましたので共有します！

- 12/6の11:30で作成されていたスパムカレンダー
    - 削除済み
- それ以降に作成されたスパムカレンダー
    - まだ残っている
    - 一括で削除やシステム的に完全に防ぐ手立てが無いため、適宜削除を行う

https://github.com/increments/qiita-discussions/discussions/76 こちらのDiscussionでも話題に出させていただいたのですが、苛烈イタチごっこでして……
抜本的な解決を目指してはいるのですがなかなか実現できずにいます
ご理解いただけますと幸いです 🙇

**Reply by simon-ritchie** [reply]
@kyntk さん @xrxoxcxox さん

ご共有＆ご対応ありがとうございます・・・！
委細承知いたしました！
